painttiming.bs

```diff
@@ -186,7 +186,9 @@ Reporting paint timing {#sec-reporting-paint-timing}
 
     1. If this instance of [=update the rendering=] is the [=first contentful paint=] of |document|, then invoke the [[#report-paint-timing]] algorithm with |document|, <code>"first-contentful-paint"</code>, and |paintTimestamp| as arguments.
 
-        NOTE: A parent frame should not be aware of the paint events from its child iframes, and vice versa. This means that a frame that contains just iframes will have [=first paint=] (due to the enclosing boxes of the iframes) but no [=first contentful paint=].
+        NOTE: A parent frame should not be aware of the paint events from its child iframes, and vice versa. This means that a frame that contains just iframes, will have [=first paint=] (due to the enclosing boxes of the iframes) but no [=first contentful paint=].
```

Comment:
Looks like you accidentally added a comma here after `contains just iframes` (there shouldn't be one).

Reply:
oops! fixing